Source/Shaders/Builtin/Functions/alphaWeight.glsl

```diff
@@ -1,32 +1,10 @@
 /**
  * @private
  */
-float czm_alphaWeight(float a)
-{
-    float x = 2.0 * (gl_FragCoord.x - czm_viewport.x) / czm_viewport.z - 1.0;
-    float y = 2.0 * (gl_FragCoord.y - czm_viewport.y) / czm_viewport.w - 1.0;
+float czm_alphaWeight(float a) {
```

Comment:
Our GLSL code style differs from our JS style, and we actually leave the curly brace on its own line.